ci/tests/viz/rpc-viz/src/entity/test.rs

```diff
@@ -9,13 +12,136 @@ fn test_to_from_proto() {
     let proto = entity.to_proto();
 
     match IdEntity::from_proto(&proto) {
-        Err(err) => {
-            panic!(
-                "Could not decode proto just created: {}. Proto: {:?}",
-                err,
-                proto
-            )
-        }
+        Err(err) => panic!(
+            "Could not decode proto just created: {}. Proto: {:?}",
+            err,
+            proto
+        ),
         Ok(got) => assert!(entity.fuzzy_eq(&got, FUZZY_EQ_THRESH)),
     }
 }
+// THIS TEST SHOULD NOT BE DELETED - COMMENTED OUT UNTIL I ADD BROWSER HANDLING
```

Comment:
I'd prefer to not merge a giant block of commented code, any chance you could put this test into a new branch and come back to it when you add the functionality to the browser?

Reply:
Good idea - I'll do